### PR TITLE
Add Prometheus Remote Write Exporter supporting Cortex - helper

### PR DESCRIPTION
**Pull request by huyan0**
This PR is part of a series of PRs implementing a Prometheus remote write exporter supporting Cortex. 

**Description:** This PR adds helper methods used by the exporter, including functionalities such as sanitizing strings and creating Prometheus labels from OTLP labels.

**Documentation:** 

* [Design](https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/prometheusremotewriteexporter/DESIGN.md)

cc: @huyan0 @alolita 

Request for review: @jmacd @bogdandrutu @annanay25 @gouthamve @open-telemetry/collector-approvers @open-telemetry/collector-maintainers

### CODE REVIEW

**Comment by huyan0** [comment]
Note that this PR contains changes in PR #1552 as well. The only new files added are helper.go and helper_test.go

**Comment by bogdandrutu** [comment]
@huyan0 rebase?

**Comment by huyan0** [comment]
> @huyan0 rebase?

Yeah, will do. Really appreciate the velocity and the content of your reviews. Have a great weekend : )

**Comment by bogdandrutu** [comment]
Please rebase, failures should disappear

**Comment by huyan0** [comment]
I pulled upstream master and rebased, but the code coverage of opencensusexporter/opencensus.go dropped... No clue how that happened. The helper.go that I added has 100% coverage.

**Comment by bogdandrutu** [comment]
Can you try one more time? Or is this up to date, very strange the condecovergs

File: exporter/prometheusremotewriteexporter/helper.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
+// Copyright 2020 The OpenTelemetry Authors
```
Comment (bogdandrutu) [inline exporter/prometheusremotewriteexporter/helper.go:1]: Remove the year

File: exporter/prometheusremotewriteexporter/helper_test.go [hunk 2]
```diff
@@ -0,0 +1,282 @@
+// Copyright 2020 The OpenTelemetry Authors
```
Comment (bogdandrutu) [inline exporter/prometheusremotewriteexporter/helper_test.go:1]: Remove year

File: exporter/prometheusremotewriteexporter/testutil_test.go [hunk 3]
```diff
@@ -0,0 +1,164 @@
+// Copyright 2020 The OpenTelemetry Authors
```
Comment (bogdandrutu) [inline exporter/prometheusremotewriteexporter/testutil_test.go:1]: Same thing here.

**Comment by huyan0** [comment]
rebased and made sure my code is 100% covered. Should I worry about the windows-msi test failing because install wix toolset times out?

**Comment by bogdandrutu** [comment]
Please rebase after https://github.com/open-telemetry/opentelemetry-collector/pull/1573 is merged

**Comment by jmacd** [comment]
\O/

**Pull request merged**